Resources/public/js/views/fields/ez-emailaddress-editview.js

```diff
@@ -101,6 +101,21 @@ YUI.add('ez-emailaddress-editview', function (Y) {
         _getFieldValue: function () {
             return this.get('container').one('.ez-emailaddress-input-ui input').get('value');
         },
+
+        /**
+         * Returns email address format validity
+         *
+         * @protected
+         * @method _isValidEmail
+         * @return {Boolean}
+         */
+        _isValidEmail: function () {
+            /* jshint -W101 */
+            var pattern = /^((\"[^\"\f\n\r\t\v\b]+\")|([A-Za-z0-9_\!\#\$\%\&\'\*\+\-\~\/\^\`\|\{\}]+(\.[A-Za-z0-9_\!\#\$\%\&\'\*\+\-\~\/\^\`\|\{\}]+)*))@((\[(((25[0-5])|(2[0-4][0-9])|([0-1]?[0-9]?[0-9]))\.((25[0-5])|(2[0-4][0-9])|([0-1]?[0-9]?[0-9]))\.((25[0-5])|(2[0-4][0-9])|([0-1]?[0-9]?[0-9]))\.((25[0-5])|(2[0-4][0-9])|([0-1]?[0-9]?[0-9])))\])|(((25[0-5])|(2[0-4][0-9])|([0-1]?[0-9]?[0-9]))\.((25[0-5])|(2[0-4][0-9])|([0-1]?[0-9]?[0-9]))\.((25[0-5])|(2[0-4][0-9])|([0-1]?[0-9]?[0-9]))\.((25[0-5])|(2[0-4][0-9])|([0-1]?[0-9]?[0-9])))|((([A-Za-z0-9\-])+\.)+[A-Za-z\-]{2,}))$/;
```

Comment:
you should put that in a constant like `VALIDATION_PATTERN` so that it's no recompiled each time this method is called.